README.md

```diff
@@ -22,12 +22,29 @@ urlFragment: uwp-signing-in-graph-aad
 
 ## About this sample
 
+### Table of contents
+- [About this sample](#about-this-sample)
+  - [Overview](#overview)
+- [Steps to run](#steps-to-run)
+- [How to run this sample](#how-to-run-this-sample)
+  - [Step 1:  Clone or download this repository](#step-1-clone-or-download-this-repository)
+  - [Step 2:  Register the sample application with your Azure Active Directory tenant](#step-2-register-the-sample-application-with-your-azure-active-directory-tenant)
+  - [Step 4: Run the sample](#step-4-run-the-sample)

```

Comment:
```suggestion
  - [Step 3: Run the sample](#step-3-run-the-sample)

```

Reply:
Updated Readme to contain Step 3.
Thanks!